packages/holocron-dev-server/lib/server.js

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import express from 'express';
+
+import {
+  loadWebpackMiddleware,
+  createRenderingMiddleware,
+  createModulesProxyRelayMiddleware,
+  createMocksMiddleware,
+  requestAcceptedMiddleware,
+} from './middleware';
+import {
+  createModuleMap,
+  getPublicUrl,
+  getStaticPath,
+  loadStatics,
+  loadLanguagePacks,
+  openBrowser,
+} from './utils';
+import {
+  setLogLevel,
+  logHotReloadReady,
+  logServerUrl,
+  logExternalsBundleAnalyzerUrl,
+  logModuleBundlerAnalyzerUrl,
+  logServerStart,
+} from './utils/logs';
+import { errorReportingUrlFragment } from './constants';
+
+export function onLaunch({
+  openWhenReady, externals = [], serverAddress, port,
+}) {
+  return (error) => {
+    if (error) throw error;
+    logServerUrl(serverAddress, port);
+    logHotReloadReady();
+    logModuleBundlerAnalyzerUrl(serverAddress);
+    if (externals.length > 0) {
+      logExternalsBundleAnalyzerUrl(serverAddress, externals);
```

Comment:
same concern with this, outside of debugging the this tool does the externals analyzer provide information to help the user make better production code.

Reply:
if we we do enable production builds, this can be something we can adopt and highlight

Reply:
It does offer insight into the size and composition of externals being used without any development code (it's production ready DLL). This ensures the accuracy across dev/prod with the externals analysis only.

Reply:
i'm still not sure how this helps users and won't add to confusion, there are many users who are already confused between the development bundle and production.